rest-api/core-rest/src/main/webapp/lib/gaffer.js

```diff
@@ -208,6 +211,27 @@ function updateDocUrl(properties) {
     });
 }
 
+function updateLogoLink(properties) {
+    updateElementWithId('logo', 'gaffer.properties.app.logo.link', properties, function(value, id) {
+        $('#' + id).attr("href", value);
+    });
+}
+
+function updateLogo(properties) {
+    updateElementWithId('logo', 'gaffer.properties.app.logo.src', properties, function(value, id) {
+        $('#' + id).prepend("<img class='logo__img' alt='swagger' height='30' width='30' src='" + value + "'/>");
```

Comment:
The Gaffer logo you have as a default is not square so setting height and width to 30 squashes it. Perhaps you should adjust the logo so it is square.